core/src/test/java/org/pentaho/platform/util/StringUtilTest.java

```diff
@@ -75,14 +92,20 @@ public void testDoesPathContainParentPathSegment() {
     }
   }
 
-  public static void main( final String[] args ) {
-    StringUtilTest test = new StringUtilTest();
+  @Test
+  public void testMain() {
     try {
-      test.testTokenStringToArray();
```

Comment:
Yes - following the preferred convention, unit tests should use annotations and not extend the TestCase class. For that reason, the main method is not needed and the class cannot be added to the test quite within TestAll, because addTestSuite accepts a TestCase argument.